tests/pr_test.go

```diff
@@ -1,60 +1,38 @@
-// Tests in this file are run in the PR pipeline and the continuous testing pipeline
 package test
 
 import (
+	"log"
 	"testing"
 
-	"github.com/stretchr/testify/assert"
-	"github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper/testhelper"
+	"github.com/gruntwork-io/terratest/modules/terraform"
 )
 
-// Use existing resource group
-const resourceGroup = "geretain-test-resources"
+const fullyConfigurableTerraformDir = "./../solutions/fully-configurable"
 
-// Ensure every example directory has a corresponding test
-const advancedExampleDir = "examples/advanced"
-const basicExampleDir = "examples/basic"
-
-func setupOptions(t *testing.T, prefix string, dir string) *testhelper.TestOptions {
-	options := testhelper.TestOptionsDefaultWithVars(&testhelper.TestOptions{
-		Testing:       t,
-		TerraformDir:  dir,
-		Prefix:        prefix,
-		ResourceGroup: resourceGroup,
-	})
-	return options
-}
-
-// Consistency test for the basic example
-func TestRunBasicExample(t *testing.T) {
-	t.Parallel()
-
-	options := setupOptions(t, "mod-template-basic", basicExampleDir)
-
-	output, err := options.RunTestConsistency()
-	assert.Nil(t, err, "This should not have errored")
-	assert.NotNil(t, output, "Expected some output")
+type TerraformTestConfig struct {
+	TerraformDir string
+	Vars         map[string]interface{}
 }
 
-func TestRunAdvancedExample(t *testing.T) {
-	t.Parallel()
-
-	options := setupOptions(t, "mod-template-adv", advancedExampleDir)
-
-	output, err := options.RunTestConsistency()
-	assert.Nil(t, err, "This should not have errored")
-	assert.NotNil(t, output, "Expected some output")
-}
-
-// Upgrade test (using advanced example)
-func TestRunUpgradeExample(t *testing.T) {
+func TestTerraformCompleteTest(t *testing.T) {
```

Comment:
Why are we not using our own test wrapper so it does consistency check etc?

Reply:
Made the change.